website/package.json

```diff
@@ -48,9 +50,12 @@
     "autoprefixer": "^10.4.20",
     "eslint": "^9.17.0",
     "hot-hook": "^0.4.0",
+    "husky": "^9.1.7",
+    "lint-staged": "^15.2.11",
     "openapi-types": "^12.1.3",
     "pino-pretty": "^13.0.0",
     "prettier": "^3.4.2",
+    "prettier-edgejs": "^0.2.34",
```

Comment:
This one is already included in `@adonisjs/prettier-config`

Reply:
Removing it from the dependencies causes this error:
```
Error: Cannot find package 'prettier-edgejs' imported from /home/ricky/Desktop/NIAEFEUP/enei/website/noop.js
    at __node_internal_ (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:15758:11)
    at new NodeError (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:15723:5)
    at packageResolve (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:16664:9)
    at moduleResolve (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:16704:18)
    at defaultResolve (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:16795:16)
    at resolve2 (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:16812:12)
    at importFromFile (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:16827:16)
    at importFromDirectory (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:21693:10)
    at importPlugin (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:21705:12)
    at async loadPluginWithoutCache (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:21709:18)
    at async Promise.all (index 0)
    at async Promise.all (index 1)
    at async Module.<anonymous> (file:///home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/prettier@3.4.2/node_modules/prettier/index.mjs:21991:17)
    at async /home/ricky/Desktop/NIAEFEUP/enei/website/node_modules/.pnpm/eslint-plugin-prettier@5.2.1_eslint-config-prettier@9.1.0_eslint@9.17.0_jiti@1.21.6___eslint@_smc4lxph37ea4e7syguhj5l7sy/node_modules/eslint-plugin-prettier/worker.js:45:41
```

Reply:
Did you run `pnpm install` after removing it from the dependencies?

Reply:
Yes

Reply:
It now works after I deleted the node_modules folder and reinstalled everything, weird